src/js/shield_defs.js

```diff
@@ -3687,6 +3687,20 @@ export function loadShields() {
     },
   };
 
+  // Saudi Arabia
+  shields["SA:national"] = {
```

Comment:
> Currently, [only one national route](https://www.openstreetmap.org/relation/11366015) is mapped with `network=SA:national`, and all other routes are missing either the `network=*` tag or a relation altogether. But `SA:national` clearly represents a coherent network, and this could encourage local mappers to complete the tags for other routes.

The one `network=SA:national` route is what Saudi Arabia calls a “highway” route, as opposed to a “city road” or “branch road”. All three classifications are part of a nationwide system. Part of Highway 40 has been mapped redundantly as [`network=SA:highway`](https://www.openstreetmap.org/relation/8249853). If we could get a local mapper to confirm their preference, that might also have the effect of encouraging more build-out of these route relations.